src/constants.js

```diff
@@ -9,3 +9,4 @@ export const TX_EXPLORER = 'https://explorer.v.systems/transactions/'
 export const FEEDBACK_URL = 'https://github.com/virtualeconomy/v-wallet-gui/issues/new'
 export const SHOW_UNSUPPORTED_FUNCTION = false
 export const MAX_ALIAS_LENGTH = 8
+export const VSYS_RATE = 'https://test-rate.virtualeconomytech.com/api/superNodesDetail'
```

Comment:
Move all APIs setting to `network.js`. The default url should use production url (domain is vsysrate.com).